profile/src/main/java/com/scottlogic/deg/profile/v0_1/SupportedVersionChecker.java

```diff
@@ -28,11 +28,17 @@
 public class SupportedVersionChecker implements SchemaVersionValidator {
     private SchemaVersionRetriever schemaVersionRetriever;
     private ProfileConfigSource configSource;
+    private SupportedVersionsGetter supportedVersionsGetter;
```

Comment:
Very minor (and I'm very open to discussion on it), but it looks like we're using `Retriever` as our naming convention here instead of `Getter`.